fenix.json.example

```diff
@@ -22,9 +22,7 @@
     "feed_info": {
         "publisher_name": "Torsten Grote",
         "publisher_url": "https://transportr.grobox.de",
-        "version":  "0.1",
-        "start_date": "20160901",
-        "end_date": "20170831"
```

Comment:
Nice that this removed here, because I prefer using the dynamic dates. However, I am a bit worried that this feature of manually specifying the dates is now not documented anymore. Do you have an idea where this could be done?